html/js/product-preferences.js

```diff
@@ -23,32 +24,18 @@ function get_user_product_preferences() {
 	return user_product_preferences;
 }
 
-function getFlavorFromHostname() { // get the flavor by the url 
-    var host = window.location.hostname;
+var type_product;
 
-    if (host.includes("openbeautyfacts")) {
-        return "obf";
-    }
-    if (host.includes("openfoodfacts")) {
-        return "off";
-    }
-    
-    return "off"; // default 
-}
-
-var flavor = getFlavorFromHostname();
-
-var reset_message;
-
-if (flavor === "off") { // get the reset message depend on the flavor
-    reset_message = lang().reset_preferences_details;
+if (flavor === "off") { // get the type of product
```

Comment:
Instead of passing the flavor to the script, you can directly pass product_type, that way you don't need this logic.